### ISSUE TITLE
Change the highlight library?

### ISSUE THREAD

**Issue by beyondghx** [tuchk4/storybook-readme#140]
I know you are highlighting the code block with highlight.js. But this library was not support well with JSX syntax. So I think if we can change the highlight library would be better(Prismjs support JSX well). Of course, highlight.js support more themes, but I think support JSX is more important.

**Comment by tuchk4** [comment]
If it will not be so hard to replace highlightjs with prism I do it in `5.0.2`

**Comment by joepuzzo** [comment]
I think an easy mean time solution would be a flag to disable the theme all together and let us theme it with prism... 

```
readme: {
    /**
     * Override theme values
     *
     * All theme values https://github.com/tuchk4/storybook-readme/blob/master/packages/storybook-readme/src/styles/githubMarkdownCss.js#L436
     */
    //theme: themes.dark,

    /**
     * Highlightjs code theme
     * Import theme at _.storybook/config.js_.
     * Full list of theme https://highlightjs.org/static/demo/.
     */
    codeTheme: null, <<< SOMTHING LIKE THIS to disable theming
  },
```

**Comment by tuchk4** [comment]
I opened PR with 100% working code switched to Prism.
There is one problem -- themes.

Here is the official github colors theme taken from here https://github.com/PrismJS/prism-themes

![image](https://user-images.githubusercontent.com/5140611/55764351-c850fb00-5a73-11e9-9916-4da50bbb8327.png)

**Comment by tuchk4** [comment]
Tomorrow will check it again, update docs and going to release new version with all features/bugs listed at #153 153

**Comment by tuchk4** [comment]
Update CSS for prism `github` color theme.
Now looks awesome

![image](https://user-images.githubusercontent.com/5140611/55792958-738c9f00-5aca-11e9-8f9a-c80412180b14.png)

**Comment by tuchk4** [comment]
published 5.0.2
